react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -128,24 +204,30 @@ type Props = {
      */
     dispatch: Function,
 
-    /**
-     * Indicates whether the thumbnail is hovered or not.
-     */
-    isHovered: ?boolean,
-
     /**
      * The ID of the participant related to the thumbnail.
      */
     participantID: ?string
-};
+|};
+
+/**
+ * Click handler.
+ *
+ * @param {SyntheticEvent} event - The click event.
+ * @returns {void}
+ */
+function onClick(event) {
+    // If the event is propagated to the thumbnail container the participant will be pinned. That's why the propagation
```

Comment:
Ditto.

Reply:
Check my previous comment. Here I don't have a suggestion for future improvement since the display name is part of the Thumbnail... 

But again I don't think we can avoid it with `pointer-events: none` because the problem is actually with the propagation of the event not with actual event triggered by a `div` that we don't want to trigger anything.

Reply:
Ok